samples/meeting-av-filters/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-meeting-av-filters-csharp",
+    "source": "officeDev",
+    "title": "Meeting av filter",
+    "shortDescription": "This sample illustrates usage of av filters for teams meetings.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/meeting-av-filters/csharp",
+    "longDescription": [
+      "This sample illustrates usage of av filters for teams meetings."
+    ],
+    "creationDateTime": "2022-08-30",
+    "updateDateTime": "2022-08-30",
+    "products": [
+      "Teams"-
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "csharp"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "netframework"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "av-filter"
```

Comment:
Teams features should be the capability you have configured in  your manifest like Tab, bot etc.
I think, you can add "**devicePermissions**"

Reply:
In other samples also using device permissions, **devicePermissions** is not added as an value in sample.json. Also av-filter itself is an capability like a bot and tab.